pytest_plugins/metadata_markers.py

```diff
@@ -154,6 +173,14 @@ def pytest_collection_modifyitems(session, items, config):
                 )
                 deselected.append(item)
                 continue
+            BaseOS_marker = item.get_closest_marker('BaseOS').args[0]
+            if BaseOS and BaseOS_marker not in BaseOS:
+                logger.debug(
+                    f'Deselected test {item.nodeid} due to "--BaseOS {BaseOS}",'
+                    f'test has BaseOS mark: {BaseOS_marker}'
+                )
+                deselected.append(item)
+                continue
```

Comment:
Not required since we are not filtering tests based on satellite BaseOS as per the above comment!

Please remove!